app/views/custom/polls/show.html.erb

```diff
@@ -31,70 +31,84 @@
         <% end %>
       <% end %>
 
+      <% if @poll.answers_descriptions_link_text.present? &&
+            @poll_questions_answers.select(&:has_more_information?).any? %>
+        <div class="callout secondary">
+          <%= @poll.answers_descriptions_link_text %><br>
+          <%= link_to t("polls.show.answers_descriptions_link_text"), "#answers_decriptions_title" %>
```

Comment:
Typo: it's missing an "s" in the middle of "decriptions" (here and in the `<h2>` it's linking to).